src/components/Urlsnarf/Urlsnarf.tsx

```diff
@@ -1,44 +1,57 @@
 import { Button, LoadingOverlay, NativeSelect, Stack, TextInput } from "@mantine/core";
```

Comment:
Remove LoadingOverlay because you are not using it. 
```suggestion
import { Button, NativeSelect, Stack, TextInput } from "@mantine/core";
```